tests/base.py

```diff
@@ -195,6 +195,10 @@ def create_connection(self, original_properties: bool = True):
         menagerie.verify_check_exit_status(self, exit_status, check_job_name)
         return conn_id
 
+    def ensure_connection(self, original=True):
+        conn_id = connections.ensure_connection(self, original_properties = original)
```

Comment:
What is the benefit of this method? Is it simply to make original a default true value? Why not fix that in the connections.py file inside of tap tester?